functests/sctp/sctp.go

```diff
@@ -0,0 +1,219 @@
+package sctp
+
+import (
+	"io/ioutil"
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/client"
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/namespaces"
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	mcfgScheme "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned/scheme"
+
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const mcYaml = "../sctp/sctp_module_mc.yaml"
+const hostnameLabel = "kubernetes.io/hostname"
+const testNamespace = "sctptest"
+
+var testerImage string
+
+func init() {
+	testerImage = os.Getenv("SCTPTEST_IMAGE")
+	if testerImage == "" {
+		testerImage = "fedepaol/sctptest:v1.1"
```

Comment:
Can you put the full path? (quay.io/docker.io)

Reply:
It's currently on dockerhub.
I have one task open to move it away from it (and ideally on a repo under our org)
I'll leave at it is for this PR